configure.ac

```diff
@@ -154,3 +154,18 @@ AC_CONFIG_FILES([Makefile
                  ublksrv.pc])
 
 AC_OUTPUT
+
+dnl Summarise version and how it is configured.
+echo
+echo "--------------------------------------------------"
+echo
+echo "               $PACKAGE_NAME   $PACKAGE_VERSION"
+echo
+AS_ECHO_N("    gnutls ................. ")
+if test "x$HAVE_GNUTLS" = "x1"; then echo "yes"; else echo "no"; fi
+AS_ECHO_N("    large file support ..... ")
+if test "$NBD_LFS" -eq "1"; then echo "yes"; else echo "no"; fi
```

Comment:
Hello Richard,

The Summary log is very nice, but not sure if it is good to add the two nbd specific features log here.

Thanks,

Reply:
My intent was to list all optional features (but actually I missed SDP).  The reason for showing them is it makes it easy to see if you're building a full-featured package -- is "yes" present on every line.